### PR TITLE
scrollbar brought back for overflowing cards

### PR DESCRIPTION
**Pull request by gemantzu**
At some point we lost the ability to scroll company cards which overflow the cards. This fixes it.
@tajchumber / @burden thoughts on improving this? maybe style a bit the scrollbar?

### CODE REVIEW

**Comment by gemantzu** [comment]
I just copied and pasted the gist, because Idk enough to tinker with it. I can either remove it and let you do the proper work, or you can do your thing after this is merged :)

**Comment by doomspork** [comment]
@tajchumber and @burden, can you take another look?

File: assets/css/app.scss [hunk 1]
```diff
@@ -30,6 +30,7 @@ html {
   background-attachment: fixed, fixed;
   background-image: asset_url("overlay.png"), asset_url("paper_fibers.png");
 
+
```
Comment (tajchumber) [inline assets/css/app.scss]: All looks good to me! Could remove the extra line here please?

File: assets/css/app.scss [hunk 2]
```diff
@@ -98,10 +99,36 @@ html {
   }
 
   .box {
+
+
```
Comment (tajchumber) [inline assets/css/app.scss]: en here as well

**Comment by tajchumber** [comment]
thanks @gemantzu !

**Pull request merged**